private/src/styles/base/_type.scss

```diff
@@ -26,31 +10,10 @@ p.has-drop-cap:not(:focus)::first-letter {
   margin-bottom: 30px;
 }
 
-p.has-small-font-size {
-  font-size: .9rem;
-}
-
```

Comment:
Gutenberg creates the `has-x-font-size` classes, so we shouldn't need these anymore if all of the required font sizes are defined in theme.json.